aiida_registry/fetch_metadata.py

```diff
@@ -192,6 +209,14 @@ def complete_plugin_data(
                     plugin_data["aiida_version"] = data.aiida_version
                     plugin_data["entry_points"] = data.entry_points
 
+    current_version = get_last_fetched_version(plugin_data['name'])
+    if current_version:
+        try:
+            if current_version != plugin_data["metadata"]["version"]:
+                plugin_data["metadata"]["release_date"] = datetime.today().strftime("%d-%m-%Y")
```

Comment:
and I guess you will manually add an initial seed of the "release_date" in the current set of plugins?